src/messages/rdf.js

```diff
@@ -0,0 +1,41 @@
+import { gettext as _, singleLineString } from 'utils';
+
+
+export const TAG_NOT_ALLOWED_HIDDEN = {
+  code: 'TAG_NOT_ALLOWED_HIDDEN',
+  message: _('The <hidden> tag is not allowed'),
+  description: _(singleLineString`Your RDF file contains the <hidden> tag,
+    which is not allowed in an Add-on.`),
+};
+
+// NOTE: I didn't want to change the commonality of messages without discussion,
```

Comment:
I can do this quickly before we merge this if you want; let me know what you think of it.

I imagine for something like RDF where we have a bunch of restricted identifiers easily found it might be the way to go. Possibly down the road with JS too.